surrogates/ga.js

```diff
@@ -45,4 +45,23 @@
         gaqObj.push(commandQueue.shift());
     }
     window._gaq = gaqObj;
+
+    /* This Source Code Form is subject to the terms of the Mozilla Public
+     * License, v. 2.0. If a copy of the MPL was not distributed with this
+     * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+    // gaplugin wrapper based on Mozilla's Google analytics shim:
+    // https://searchfox.org/mozilla-central/source/browser/extensions/webcompat/shims/google-analytics-and-tag-manager.js
+    if (!window?.gaplugins?.Linker) {
```

Comment:
is it safe for us to use fancy `?.` syntax for surrogates?

Reply:
I hope so, because we use much fancier syntax in content-scope-scripts. It certainly is OK for the browser extension, but I don't know how to verify it for all of our browsers though. I'll just rewrite it to be safe, we don't benefit from taking a risk.

Reply:
Done.